.gitignore

```diff
@@ -0,0 +1,45 @@
+*.d
```

Comment:
If you wanted a gitignore for elements with native code - you have missed entries from the old gitignore (before force pushing, based on the other repo I guess). Plus stripping comments from the generated gitignore seems like a bad idea, previously it was clear what's part of the template and what is custom.
However, I don't think we're going to need native code here, you can stick to what you've used in opus_format